packages/components/src/internal/components/entities/EntityInsertPanel.tsx

```diff
@@ -422,7 +422,8 @@ export class EntityInsertPanelImpl extends Component<Props, StateProps> {
                                 queryModel,
                                 new EditorModel({ id: ENTITY_GRID_ID }),
                                 new EntityGridLoader(insertModel, queryModel.queryInfo),
-                                queryModel
+                                queryModel,
+                                this.isIncludedColumn
```

Comment:
Hmm, I'd like to better understand how #1029 caused this regression. I agree that it did, however, I don't see where it was filtering for the `this.isIncludedColumn` previously? I had removed a usage of it upon submission. Is that the one?

Thanks for fixing this.

Reply:
Yep, it was needed in insertModel.postEntityGrid